front/src/common/IntervalsDataViz/data.ts

```diff
@@ -238,6 +238,9 @@ export function fixLinearMetadataItems<T>(
       } as LinearMetadataItem<T>,
     ];
   }
+  const filteredItems = items
+    .filter((item) => item.begin < lineLength)
+    .map((item) => (item.end >= lineLength ? { ...item, end: lineLength } : item));
```

Comment:
Use the function you defined in the utils files
```suggestion
  const filteredItems = removeInvalidRanges(items, lineLength);
```